### PR TITLE
Ci dev

### LINKED ISSUE #45: Install script
prepare (one) script that will install kontext locally

### PR DESCRIPTION
**Pull request by vidiecan**
fixed #45

### CODE REVIEW

**Comment by kosarko** [comment]
@vidiecan two questions what is the first huge squashed commit? Is that just syncing with upstream or are there our changes? Second how does this play with #49 / #56 both add `.travis.yml`

**Comment by vidiecan** [comment]
the squashed is a bit

**Comment by vidiecan** [comment]
the big squashed, well, those are 20+ testing commits squashed and some rebased ones got into the way... should not be a big deal when merging

**Comment by vidiecan** [comment]
it does not play well, .travis.yml must be merged in such a way that there will be two jobs running - one for python mocked unittests, the other one for a full installation + tests (in future also for `real world` unittests)

**Comment by kosarko** [comment]
> it does not play well, .travis.yml must be merged in such a way that there will be two jobs running - one for python mocked unittests, the other one for a full installation + tests (in future also for real world unittests)

ok so a "normal" conflict resolution, we'll see what ends up in kontext-dev first :)

File: build-scripts/setup.sh [hunk 1]
```diff
@@ -0,0 +1,297 @@
...
+
+if [[ "x$MYSQLUSER" == "x" ]]; then
+    export MYSQLUSER=root
+fi
+if [[ "x$MYSQLPASS" == "x" ]]; then
+    export MYSQLPASS=
+fi
+
+mkdir -p ${DEPSDIR}
+sudo chown -R ${USER}:${USER} /opt/kontext
```
Comment (kosarko) [inline build-scripts/setup.sh]: I guess this and any other instance of `/opt/kontext` should be `${KONTEXTDIR}` or some other variable

Comment (vidiecan) [inline build-scripts/setup.sh]: KONTEXT_PREFIX
I use KONTEXTDIR to point to a shared directory with kontext sources that is away from KONTEXT_PREFIX

**Pull request merged**